src/config.rs

```diff
@@ -2425,6 +2425,12 @@ pub fn persist_config(config: &TiKvConfig) -> Result<(), String> {
     let last_cfg_path = store_path.join(LAST_CONFIG_FILE);
     let tmp_cfg_path = store_path.join(TMP_CONFIG_FILE);
 
+    if let Some(last_cfg) = get_last_config(&config.storage.data_dir) {
+        if serde_json::to_string(&config).unwrap() == serde_json::to_string(&last_cfg).unwrap() {
```

Comment:
How about:
```
let same = fs::read_to_string(&last_cfg_path).map_or(false, |last_cfg| {
    serde_json::to_string(&config).unwrap() == last_cfg
});
if same {
    return Ok(());
}
```

Reply:
@BusyJay Nice! I'd wondered about how to read last config by `tmp_cfg_path`. And it's more readable to declare a   variable `same` first.